### PR TITLE
Update substrate

### PR DESCRIPTION
**Pull request by gaudenzkessler**
update substrate sp-core

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -11,8 +11,8 @@ members = [
 sgx_tstd = { version = "1.1.4", git = "https://github.com/haerdib/incubator-teaclave-sgx-sdk", branch = "master"}
 sgx_types = { version = "1.1.4", git = "https://github.com/haerdib/incubator-teaclave-sgx-sdk", branch = "master"}
 
-#[patch."https://github.com/integritee-network/pallets.git"]
-#pallet-parentchain = { path = '../pallets/parentchain' }
+#[patch."ttps://github.com/integritee-network/pallets.git"]
```
Comment (haerdib) [inline Cargo.toml]: I don't think that's a good change 😄

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -47,6 +47,14 @@ sp-version = { version = "4.0.0-dev", default-features = false, git = "https://g
 # Integritee dependencies
 pallet-parentchain = { default-features = false, git = "https://github.com/integritee-network/pallets.git", branch = "master" }
 
+# Ajuna dependencies
+pallet-ajuna-connectfour = { default-features = false, git = "https://github.com/ajuna-network/ajuna-node.git", branch = "darkfriend77/integritee" }
+pallet-scheduler = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
+
+#[patch."https://github.com/ajuna-network/ajuna-node.git"]
```
Comment (haerdib) [inline runtime/Cargo.toml]: Can you remove these comments? They are already in the top most .toml

File: runtime/Cargo.toml [hunk 3]
```diff
@@ -69,6 +77,7 @@ std = [
     'frame-system-rpc-runtime-api/std',
     'pallet-aura/std',
     'pallet-balances/std',
+    'pallet-ajuna-connectfour/std',
```
Comment (haerdib) [inline runtime/Cargo.toml]: Two remarks:
- ordering
- pallet-scheduler is missing I think

File: test-no-std/Cargo.toml [hunk 4]
```diff
@@ -5,28 +5,11 @@ authors = ["Integritee AG <hello@integritee.network>"]
...
-branch = "master"
-default-features = false
-features = ["full_crypto"]
-
-[dependencies.sp-core]
-git = "https://github.com/paritytech/substrate.git"
-branch = "master"
-default-features = false
-features = ["full_crypto"]
+libc = { version = "0.2", default-features = false }
```
Comment (haerdib) [inline test-no-std/Cargo.toml:11]: Much nicer than before 👍

File: Cargo.toml (hunk 1 above)
Comment (gaudenzkessler) [reply Cargo.toml]: you thing we shouldnt take the ttps-protocol? ;)

**Review by darkfriend77 — APPROVED** [review]
nice.

**Pull request merged**